### DISCUSSION TITLE
[Feature] set timezone for log

### DISCUSSION THREAD

**Discussion by AntonioSun** [anasoid/docker-jmeter#58] [Ideas]
Wish: able to set timezone so that logs can show correct local time:

![image](https://user-images.githubusercontent.com/2840074/150609596-560df691-f058-43a0-8e77-bb943a5d63c2.png)

E.g., `echo America/Toronto > /etc/timezone`

```
$ cat /etc/timezone 
America/Toronto
```

thx

**Comment by anasoid** [comment]
hello   @AntonioSun ,

Please create an issue, i will resolve  it tomorrow

**Comment by anasoid** [comment]
Hello @AntonioSun 
you can see how to change timezone in documentation : https://github.com/anasoid/docker-jmeter#timezone
